tools/ipapy2/macros.xml

```diff
@@ -0,0 +1,39 @@
+<macros>
+    <token name="@TOOL_VERSION@">1.3.0</token>
+
+    <xml name="citations">
+        <citations>
+        <citation type="doi">10.1021/acs.analchem.9b02354</citation>
+        <citation type="doi">10.1093/bioinformatics/btad455</citation>
+        </citations>
+    </xml>
+
+    <xml name="ionisation">
+        <param label="ionisation" name="ionisation" type="select" help="ionisation. positive +1 or negative -1">
+            <option value="1">positive</option>
+            <option value="-1">negative</option>
+        </param>
+    </xml>
+
+    <xml name="bio_connections">
+        <param label="connections list provided" name="connection_list" type="select" optional="true" multiple="true" display="checkboxes">
+            <help>list of possible connections between compounds defined as formulas. Only necessary if mode='connections'.</help>
+            <option value="C3H5NO">C3H5NO</option>
+            <option value="C6H12N4O">C6H12N4O</option>
+            <option value="C4H6N2O2">C4H6N2O2</option>
+        </param>
+    </xml>
```

Comment:
This list should probably contain all possible reactions or combinations and some background about what they are (methylation, whatever).

Reply:
Surprisingly, this macro is never used. I would look further into it